source/agora/node/Config.d

```diff
@@ -174,6 +174,9 @@ public struct InterfaceConfig
     /// Bind port
     public ushort port;
 
+    /// Proxy Protocol V1
+    public bool proxy_proto;
```

Comment:
We probably could get away without configuring this explicitly. `RPCConnection` could easily eavesdrop on the reads and extract any ProxyProtocol data that may be in the buffers. Overriding `read` & `remoteAddress` could be enough IMO.

but like you said, this should be part of vibe.d not in agora. Do we need this immediately? maybe we can hold on until we can implement this in vibed.

Reply:
Protocol states
```
The receiver MUST be configured to only receive the protocol described in this
specification and MUST not try to guess whether the protocol header is present
or not.
```

Reply:
Rebase, than we can merge

Reply:
Rebased